packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -717,6 +727,7 @@ class _TextSelectionGestureDetectorState extends State<TextSelectionGestureDetec
     return GestureDetector(
       onTapDown: _handleTapDown,
       onTapUp: _handleTapUp,
+      onForcePressStart: _forcePressStarted,
```

Comment:
Doesn't the long press recognizer and the force touch recognizer cancel each other out? Or was the original issue incorrect and it's just force touch not long press + force touch?

Reply:
For this functionality, it's just a force press. Long press + force touch does the whole moving the text thing, which was the other issue I linked to you.